### PR TITLE
fea: added text utilities 

### PR DESCRIPTION
**Pull request by nshanbhag20001412**
Moved the text related CSS from other components to text utilities

Preview : https://tx5ku.csb.app/ .Go in the docs section and click text utility

### CODE REVIEW

File: Component/Text-Utilies/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    list-style: none;
+    margin:1rem 0;
+    font-size:1.655rem;
+}
+
+a {
+    text-decoration: none;
+    cursor: pointer;
+    color: inherit;
+  }
```
Comment (gopalbbharadva) [inline Component/Text-Utilies/text-utilities.css]: Improvement: Don't apply styles on direct element name, instead use selectors(class). Or you can make default or global css file for repeating properties. Reason is that if make any new element by default this style will apply.

**Review by gopalbbharadva — APPROVED** [review]

File: Component/Text-Utilies/text-utilities.css (hunk 1 above)
Comment (nshanbhag20001412) [reply Component/Text-Utilies/text-utilities.css]: I wanted these to get applied to all the components and the user who will use the component. I also mentioned them at the end of the text utilities. Still, I plan to change it at the later stages.

**Review by adidoshi — APPROVED** [review]
Looks fine for me! Great you made separate text css utilities.

File: Component/Text-Utilies/text-utilities.css (hunk 1 above)
Comment (gopalbbharadva) [reply Component/Text-Utilies/text-utilities.css]: Ya then it's fine. I also did this and In yesterday's PR review session MAs told that dont do this that's why I told you.

**Pull request merged**